tools/build/.eslintrc.js

```diff
@@ -1,10 +0,0 @@
-/*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
-module.exports = {

```

Comment:
Didn't we keep this here for backwards compatibility?

Reply:
ah, yes, forgot about that part. I need to add a comment as a reminder.